R/symbols.R

```diff
@@ -0,0 +1,14 @@
+# Locations of symbols in TF 1.x vs TF 2.x
```

Comment:
I like centralizing these in symbols.R but you may need them to be actual R function calls rather than just resolving to the underlying TF symbol. This is because as currently stands I believe these symbols will be resolved at package build time rather than at runtime (I could be wrong about that, but that's how I'm reading it).

Reply:
I just tested by reinstalling `tensorflow` (different version) while not rebuilding `tfdatasets`, it worked fine, but I wonder does this generalize?

Reply:
In any case, to be sure, you meant something like 

```
tfr_random_uniform <- function() if (tensorflow::tf_version() < "2.0") tf$random_uniform else tf$random$uniform
tfr_random_uniform <- tfr_random_uniform()
```

i.e., initializing the variable in that same file, right? As this:

```
x + tfr_random_uniform()(shape(), -10L, 10L, tf$int64)
```

probably doesn't look so good...

Reply:
You can do it this way:

```r
tfr_random_uniform <- function(...) 
   if (tensorflow::tf_version() < "2.0") 
     tf$random_uniform(...) 
   else 
     tf$random$uniform(...)
```

You definitely can't call tf$ functions during package build time (they'll actual fail entirely when building on a system with TF, and even on one with TF they will be executed too soon -- build time vs. runtime)

Reply:
Yeah, I tried this out with a new package and there is no lazy evaluation whatsoever, so we do need to create a wrapper function like JJ suggests.